dotcom-rendering/makefile

```diff
@@ -46,13 +46,12 @@ start-ci: install gen-dotenv-ci
 	@DISABLE_LOGGING_AND_METRICS=true node dist/frontend.server.js
 
 start: export NODE_ENV=production
-start: install gen-dotenv
+start: gen-dotenv
 	$(call log, "starting PROD server...")
 	@echo '' # just a spacer
-	pm2 start dist/frontend.server.js
+	pm2 start --uid dotcom-rendering --gid frontend dist/frontend.server.js
 	@echo '' # just a spacer
 	$(call log, "PROD server is running")
-	@pm2 logs
```

Comment:
I've not found any benefit to these logs appearing locally but do we need this command on the production servers?

Reply:
This command starts streaming the logs into your console - this is unnecessary for servers as the 'console' is autonomous, and pm2 will continue to save these logs anyway in case they're needed :)